README.md

```diff
@@ -31,13 +31,24 @@ if it stops at the wrong time (the window is short), it can disable your partiti
 4. Go to `http://4.3.3.4/` in your browser. This will take you to the main menu.
 5. Click `Configure Wifi` and set up your actual wifi. The device will reboot.
 6. Find the IP of the device, and connect to it.
-7. Click `Read partitions` to see the current partition table.
+7. Click `List partitions` to see the current partition table.
```

Comment:
This says "List partitions" in the [`0.1.0`](https://github.com/softplus/Esp32Repartition/releases/tag/v0.1.0) build.